apps/builder/app/shared/style-object-model.ts

```diff
@@ -236,7 +236,7 @@ const matchKeyword = (styleValue: undefined | StyleValue, keyword: string) =>
 /**
  * stable invalid values to support caching
  */
-const guaranteedInvalidValue: StyleValue = { type: "guaranteedInvalid" };
+export const guaranteedInvalidValue: StyleValue = { type: "guaranteedInvalid" };
```

Comment:
not for export, just write an object by hand pls

Reply:
why not, its needed in many places

Reply:
type is, here it's just implementation detail we need to keep it stable in module scope. Let's please not refuse every little piece and value just because you can. Over reusability often damages huge code bases.

Reply:
alright